dockerfiles/theia/docker/ubi8/builder-clone-theia.dockerfile

```diff
@@ -1,3 +1,4 @@
 # Clone theia and keep source code in home
 RUN git clone --branch ${GIT_BRANCH_NAME} --single-branch https://github.com/${THEIA_GITHUB_REPO} ${HOME}/theia-source-code && \
     cd ${HOME}/theia-source-code && git checkout ${THEIA_COMMIT_SHA}
+RUN cd ${HOME} && tar zcf ${HOME}/theia-source-code.tgz theia-source-code
```

Comment:
why do we need that ?
https://github.com/redhat-developer/codeready-workspaces-theia/blob/crw-2-rhel-8/dockerfiles/theia/Dockerfile#L36

Reply:
I mean why did you remove it from https://github.com/redhat-developer/codeready-workspaces-theia/commit/8784e951a715a84cbdcc6037af91c44d1c935780#diff-aabad8f4a3b5d17a3038533a360278704d1160ac9d0a3ad4ff66d588f20c399d

Reply:
please don't do any tgz there, we don't need it

Reply:
I didn't remove it ... it was removed upstream and then that change was synced to downstream :) But it's OK, we can work around this downstream by overriding .dockerfile stubs.